### DISCUSSION TITLE
21.1.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1272] [General]
## Releases Notes for 21.1.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/draw.io-21.1.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/draw.io-21.1.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/draw.io-universal-21.1.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/drawio-amd64-21.1.2.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/drawio-x86_64-21.1.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/drawio-x86_64-21.1.2.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Disables compression of diagram data by default https://github.com/jgraph/drawio/issues/1724
- Updates to [draw.io core 21.1.2](https://github.com/jgraph/drawio/blob/v21.1.2/ChangeLog). All changes from 20.8.17 to 21.1.2 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.1.2'>21.1.2</a>.</em>

**Comment by rpbiz** [comment]
One of the greatest discoveries I ever made.

I first started using draw.io through domain and today found out that it has an application too for all platforms. Crazy!!!

Thank you to the team for building it.

**Comment by Naveensharmas** [comment]
![image](https://user-images.githubusercontent.com/129605394/229294749-a65ff685-525c-454f-9d19-08d07d06b612.png)

That's my first visual diagram created using draw.io online. Today I found out that it's available for windows and other platform. Thanks to the dev team who worked hard to make this happen.

**Comment by irdroid3** [comment]
What is the difference between the installers? - IE what do the options : "no-installer" and "blockmap" specifically?

**Reply by davidjgraph** [reply]
The blockmap isn't an installable version, it's just an index file that needs to be published for the updater to work.

**Comment by alvinchey** [comment]
Hi, I am getting a 1 security flag for the file https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/draw.io-21.1.2-windows-no-installer.exe at Virus Total (previous version was OK). Please advise. Thanks for great software!

**Reply by davidjgraph** [reply]
I assume the vendor has some way to flag a false positive?
